spec/models/patient_registrations_per_day_per_facility_spec.rb

```diff
@@ -31,5 +37,9 @@
     it 'should return at least one row per facility' do
       expect(query_results.pluck(:facility_id).uniq).to match_array(facilities.map(&:id))
     end
+
+    it 'counts only patients who are diagnosed hypertensive' do
+      expect(query_results.pluck(:registration_count).uniq).to eq([1])
```

Comment:
We should assert that specific patient IDs are included and excluded in this spec. This spec right now would pass if we flipped to `hypertension=no` patients by accident.

Reply:
This view returns the counts of patients, so there is no way to do ID matches. Also, the count check works since facilities would have `2 bps` on `1.day.ago` if `hypertension=no` was accidentally flipped; the spec will fail in that case.

Reply:
Ah, so this is also a consequence of the view aggregating the counts. In that case, yup, asymmetric data should do the trick 👍